README.md

```diff
@@ -72,6 +72,83 @@ The binary will be created in `./target/release/magmawm`.
 ## Install
 **MagmaWM** is still under heavy development and installation is not recommended.
 If you really want to, run the following command to install MagmaWM: 
+
+#### Nixos
+The following are two different ways to go about installing MagmaWM
+##### With overlays
+The cleaner option, but can cause issues with hash mismatching
+```nix
```

Comment:
idealy these should be markdown dropdown menus (see https://gist.github.com/citrusui/07978f14b11adada364ff901e27c7f61)